libs/types-common/src/Util/Test.hs

```diff
@@ -0,0 +1,59 @@
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+
+module Util.Test where
+
+import Data.Monoid
+import Data.Proxy
+import Data.Tagged
+import Data.Typeable
+import Options.Applicative
+import Test.Tasty
+import Test.Tasty.Options
+
+newtype ServiceConfigFile = ServiceConfigFile String
+    deriving (Eq, Ord, Typeable)
+
+instance IsOption ServiceConfigFile where
+    defaultValue = ServiceConfigFile "/etc/wire/service/conf/service.yaml"
+    parseValue = fmap ServiceConfigFile . safeRead
+    optionName = return "service-config-file"
+    optionHelp = return "Service config file to read from"
+    optionCLParser =
+      fmap ServiceConfigFile $ strOption $
+        (  long (untag (optionName :: Tagged ServiceConfigFile String))
+        <> help (untag (optionHelp :: Tagged ServiceConfigFile String))
+        )
+
+newtype IntegrationConfigFile = IntegrationConfigFile String
+    deriving (Eq, Ord, Typeable)
+
+instance IsOption IntegrationConfigFile where
+    defaultValue = IntegrationConfigFile "/etc/wire/conf/integration.yaml"
```

Comment:
The default values `"/etc/wire/conf/integration.yaml"` and `"/etc/wire/service/conf/service.yaml"` are not good, as they wouldn't match for any service (brig's service.yaml and cargohold's service.yaml (if we renamed the files that) couldn't both be under that path). Would there be a way to specify the default path per service, rather than in types-common? Or do you intend to merge all configuration files into a single `wire.yaml` (and another `integration.yaml`) ?